pattern_import_export/views/pattern_import_export.xml

```diff
@@ -45,18 +45,19 @@
                     'invisible': [('hidden_fields', 'ilike', 'pattern_export_id')]}"/>
                 <field name="related_model_id" invisible="1"/>
                 <field name="last_field_id" invisible="1"/>
-                <field name="add_select_tab" invisible="1"
+                <field name="add_select_tab"
                        attrs="{
                        'invisible': [('hidden_fields', 'ilike', 'add_select_tab')],
                        'readonly': [('hidden_fields', 'ilike', 'add_select_tab')]
                        }"
                 />
-                <field name="use_tab"
+                <field name="related_model_name" invisible="1"/>
+                <field name="tab_filter_id"
+                       domain="[('model_id', '=', related_model_name)]"
```

Comment:
instead of adding the field "related_model_name" you can directly filter on "related_model_id"

Reply:
hum in fact model_id is a model name not an id. So it's correct